apis/openstack-swift/src/test/java/org/jclouds/openstack/swift/v1/blobstore/RegionScopedSwiftBlobStoreParallelLiveTest.java

```diff
@@ -105,13 +109,40 @@ public void setup() throws IOException, InterruptedException {
    public void cleanupFiles() {
       // Delete local file
       delete(bigFile);
+      delete(smallFile);
       delete(new File(bigFile + ".downloaded"));
 
       // Delete uploaded file
       blobStore.clearContainer(CONTAINER);
       blobStore.deleteContainer(CONTAINER);
+      blobStore.clearContainer(CONTAINER_TEMP);
+      blobStore.deleteContainer(CONTAINER_TEMP);
    }
 
+   @Test(expectedExceptions = { IllegalStateException.class })
+   public void getBlobTestFailure() {
+      blobStore.containerExists(CONTAINER_TEMP);
```

Comment:
What are these tests supposed to do and why are they in the parallel class?  When I run `CloudFilesUSBlobIntegrationParallelLiveTest.getBlobTestFailure` it fails with and without your changes to `createContainerInLocation`.

Reply:
@archupsg03 Can you explain this pull request or can I close it?